bindata/etcd/restore-pod.yaml

```diff
@@ -24,6 +24,43 @@ spec:
         export ETCD_NAME=${NODE_NODE_ENVVAR_NAME_ETCD_NAME}
         export ETCD_INITIAL_CLUSTER="${ETCD_NAME}=https://${NODE_NODE_ENVVAR_NAME_ETCD_DNS_NAME}:2380"
         env | grep ETCD | grep -v NODE
+        export ETCD_NODE_PEER_URL=https://${NODE_NODE_ENVVAR_NAME_ETCD_DNS_NAME}:2380
+
+        # checking if data directory is empty, if not etcdctl restore will fail
+        if [ ! "$(ls -A /var/lib/etcd)" ]; then
+          echo "please delete the data directory before restoring"
+          exit 1
+        fi
+
+        # check if we have backup file to be restored and if the backup file is not currently
+        # being copied, exit non-zero if copying continues for more than 5 seconds
+        if [ ! -f var/lib/etcd-backup/snapshot.db ]; then
+          echo "please make a copy of the backup file, then move that copy to /var/lib/etcd-backup/snapshot.db"
```

Comment:
Currently the backup is a directory containing static resources and a snapshot db. Here using the word backup may confuse user. Call it `snapshot db` perhaps?

Reply:
@retroflexer I will update the logging, but FTR the user will never have to see that logging if restore.sh is run. The script will make sure that the file exists.